src/commands/deploy/IDeployBaseContext.ts

```diff
@@ -14,6 +14,7 @@ export interface IDeployBaseContext extends ISubscriptionActionContext {
     targetContainer?: ContainerAppModel;
 
     imageSource?: ImageSourceValues;
+    useQuickStartImage?: boolean;
```

Comment:
Are you open to using `showQuickStartImage` or similar instead of `useQuickStartImage`?  I feel like when I see the word "use" I'm expecting that we are defaulting to that choice rather that just offering the option as a pick.